gruntwork-install

```diff
@@ -230,12 +232,12 @@ function run_module {
   local readonly module_name="$1"
   local readonly download_dir="$2"
   shift 2
-  local readonly module_params=($@)
+  local -ra module_params=("$@")
   local readonly install_script_path="${download_dir}/${module_name}/${MODULE_INSTALL_FILE_NAME}"
 
   log_info "Executing $install_script_path ${module_params[@]}"
   chmod u+x "$install_script_path"
-  eval "$install_script_path ${module_params[@]}"
```

Comment:
Is eval necessary here????

Reply:
No, I don't think so. Looks like the same sort of issue as in https://github.com/gruntwork-io/module-aws-monitoring/pull/145.